Automattic/wp-calypso
# Onboarding WordPress importer: content-only flow

#### Changes proposed in this Pull Request

* Implemented WordPress - content only import flow

#### Screenshot
<img width="790" alt="Screenshot 2022-01-20 at 15 11 35" src="https://user-images.githubusercontent.com/1241413/150354889-19edcfb3-1875-4ebd-aa06-2dfe2f248e44.png">

#### Testing instructions

* Go to `/start/importer/capture?siteSlug={SLUG}`
* Enter the source website URL, example: `amused-swan.jurassic.ninja`
* From the ready screen, press the "Import your content" button
* Then select the "Content only" option by pressing the "Continue" button
* Upload your WXR file
* Check if there is a proper Progress screen with a Hooray screen at the end




Related to #59042
